docker/standalone/install_pydoop_features.sh

```diff
@@ -1,31 +1,39 @@
 set -ex
 
 . /etc/profile
+. ~/.bashrc
 
 set -u
 
-easy_install pip
+easy_install --user pip
+pip install --user avro
+pip install --user libtiff
 
-git clone https://github.com/openmicroscopy/bioformats.git
-git clone https://github.com/wnd-charm/wnd-charm.git
-git clone https://github.com/simleo/pydoop-features.git
+# clone dependencies with depth=1 to speed things up
+git clone --depth=1 --branch='metadata/merge/trigger' \
```

Comment:
I need the `metadata/merge` version, which is tagged as 5.1.8 just like the *real* 5.1.8. I have to populate the Maven cache before installing pydoop-features so that it does not download the real 5.1.8 to satisfy the dependency.